config/logstash.yml

```diff
@@ -107,6 +107,21 @@
 #
 # queue.max_events: 0
 #
+# If using queue.type: persisted, the maximum number of acked events before forcing a checkpoint
+# Default is 1024, 0 for unlimited
+#
+# queue.checkpoint.max_acks: 1024
```

Comment:
as discussed on Slack, we should just call them `queue.checkpoint.{acks, writes, interval}`